src/Test/AbstractBlockServiceTestCase.php

```diff
@@ -11,19 +11,27 @@
 
 namespace Sonata\BlockBundle\Test;
 
+use PHPUnit\Framework\TestCase;
 use Sonata\BlockBundle\Block\BlockContextInterface;
 use Sonata\BlockBundle\Block\BlockContextManager;
 use Sonata\BlockBundle\Block\BlockContextManagerInterface;
 use Sonata\BlockBundle\Block\BlockServiceInterface;
 use Sonata\BlockBundle\Block\BlockServiceManagerInterface;
 use Sonata\BlockBundle\Tests\Block\Service\FakeTemplating;
```

Comment:
Remove this use to make test pass

Reply:
Thanks for the hint buddy!

Reply:
Hmm but that breaks :P

Reply:
Oh sorry I thought you meant the `TestCase one